tests/springboot/src/main/java/io/hawt/tests/spring/boot/SpringBootService.java

```diff
@@ -41,7 +41,6 @@ public HawtioPlugin samplePlugin() {
          * - module: The path exposed from Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
          */
         HawtioPlugin plugin = new HawtioPlugin(
-            "",
```

Comment:
Now you omitted this parameter, so should update the comment above accordingly (i.e. `url`).